covidapi/api/views.py

```diff
@@ -8,6 +11,9 @@
 
 router = APIRouter()
 
+@router.get('/')
+def root():
```

Comment:
Do you need both this and the `app.mount(..., StaticFiles(...))` line below? It seems like you could do the same thing with only the `app.mount` call.

Reply:
Noted! I went ahead and got rid off the `app.mount` call and left the base route as-is.